src/components/entity/show.vue

```diff
@@ -154,7 +155,8 @@ const requestDelete = () => {
     .catch(noop);
 };
 
-const branchData = modalData();
+const EntityBranchData = defineAsyncComponent(loadAsync('EntityBranchData'));
```

Comment:
The component will only be loaded if it's rendered (if the `v-if` is `true`). So it won't ever have to be loaded for users.